cmd/aws-vpc-cni-init/main.go

```diff
@@ -125,10 +129,33 @@ func configureIPv6Settings(procSys procsyswrapper.ProcSys, primaryIF string) err
 		val, _ := procSys.Get(entry)
 		log.Infof("Updated %s to %s", entry, val)
 
+		// accept_ra must be set to 2 so that RA routes are installed by the kernel on secondary ENIs
+		// For IPv6, this setting must be inherited by the trunk ENI. It must be set here as IPAMD does
+		// not have permission to set sysctl values.
+		entry = "net/ipv6/conf/default/accept_ra"
+		err = procSys.Set(entry, "2")
+		if err != nil {
+			return errors.Wrap(err, "Failed to set IPv6 accept Router Advertisements to 2")
+		}
+		val, _ = procSys.Get(entry)
+		log.Infof("Updated %s to %s", entry, val)
+
+		entry = "net/ipv6/conf/default/accept_redirects"
+		err = procSys.Set(entry, "1")
+		if err != nil {
+			return errors.Wrap(err, "Failed to enable IPv6 accept redirects")
+		}
+		val, _ = procSys.Get(entry)
+		log.Infof("Updated %s to %s", entry, val)
+
+		// For the primary ENI in IPv6, sysctls are set to:
```

Comment:
Do we even need this block now? We're now configuring the same set of values for all interfaces on the node, so I don't see a need to repeat these again for primary IF.